flink-runtime/src/test/java/org/apache/flink/runtime/util/SerializedThrowableTest.java

```diff
@@ -179,4 +188,62 @@ void testCopySuppressed() {
                 .isInstanceOf(SerializedThrowable.class)
                 .hasMessage("java.lang.Exception: suppressed");
     }
+
+    @Test
+    void testCyclicSuppressedThrowableSerialized() {
+        SerializedThrowable serializedThrowable = new SerializedThrowable(mockThrowable());
+        assertThat(serializedThrowable).isNotNull();
+    }
+
+    // Use the repeated test for checking the thread safe and stable.
+    @RepeatedTest(16)
```

Comment:
TBH, I prefer to not introduce such repeat tests since this increase the CI time and resources.